owr/owr_bus.h

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright (c) 2015, Ericsson AB. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without modification,
+ * are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ * list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice, this
+ * list of conditions and the following disclaimer in the documentation and/or other
+ * materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ * ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+ * WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+ * IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT,
+ * INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT
+ * NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+ * PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY,
+ * WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY
+ * OF SUCH DAMAGE.
+ */
+
+/*/
+\*\ OwrBus
+/*/
+
+#ifndef __OWR_BUS_H__
+#define __OWR_BUS_H__
+
+#include "owr_message_origin.h"
+#include "owr_types.h"
+
+#include <glib-object.h>
+
+G_BEGIN_DECLS
+
+typedef enum {
+    OWR_MESSAGE_TYPE_ERROR = (1 << 0),
+    OWR_MESSAGE_TYPE_STATS = (1 << 1),
+    OWR_MESSAGE_TYPE_EVENT = (1 << 2)
+} OwrMessageType;
+
+/**
+ * OwrMessageSubType:
+ *
+ * @OWR_ERROR_TYPE_GST_ERROR: a GStreamer error message was received
```

Comment:
Media processing error or something? It's not really interesting to the application that it's specifically from GStreamer. I guess.